source/tutorials/first-steps/nix-language.md

```diff
@@ -1112,7 +1079,9 @@ Each of them is explained in the following, and here is an overview:
   ```
 
 Functions have no names.
-We say they are anonymous, and call such a function a *lambda*.
+We say they are anonymous, and call such a function a *lambda*.[^lambda]
+
+[^lambda]: The term *lambda* is a shorthand for [lambda abstraction](https://en.m.wikipedia.org/wiki/Lambda_calculus#lambdaAbstr) in the [lambda calculus](https://en.m.wikipedia.org/wiki/Lambda_calculus).
```

Comment:
Don't use the mobile links since desktop users will be greeted with a mobile version of Wikipedia.

Reply:
...which is better in so many ways, but indeed, I did that inadvertently because my browser redirects me to it automatically. Will fix.

Reply:
```suggestion
[^lambda]: The term *lambda* is a shorthand for [lambda abstraction](https://en.wikipedia.org/wiki/Lambda_calculus#lambdaAbstr) in the [lambda calculus](https://en.wikipedia.org/wiki/Lambda_calculus).
```